src/Illuminate/Http/Request.php

```diff
@@ -630,6 +630,22 @@ public static function createFromBase(SymfonyRequest $request)
 	/**
 	 * {@inheritdoc}
 	 */
+	protected function prepareBaseUrl()
+	{
+		$scriptName = $this->server->get('SCRIPT_NAME');
+		if(starts_with(ltrim($this->server->get('REQUEST_URI'), '/'), ltrim($scriptName, '/')))
```

Comment:
Where exactly? Are there any more specific style guidelines regarding spacing? (other than PSR-1 and that bit in the [docs](http://laravel.com/docs/5.0/contributions#coding-style))